### PR TITLE
Create install_NGC_PaddlePaddle_ch.rst

### PR DESCRIPTION
**Pull request by onecatcn**
one page on installing instruction on NGC PaddlePaddle was created.

### CODE REVIEW

File: docs/install/instalL_NGC_PaddlePaddle_ch.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
+..  _install_introduction:
+
+=========
+NGC PaddlePaddle 容器安装指南
```
Comment (Ligoml) [inline docs/install/instalL_NGC_PaddlePaddle_ch.rst:4]: image

请检查一下预览文件，有很多报错需要解决

**Review by onecatcn — COMMENTED** [review]
I made some changes according to the feedback. It seems resolved the issues raised in the previous check.

**Review by onecatcn — COMMENTED** [review]
update the head footnote

**Review by dingjiaweiww — APPROVED** [review]
LGTM

**Review by onecatcn — COMMENTED** [review]
remove the numbers in the headlines, in oder to make the doc align with other docs

**Review by onecatcn — COMMENTED** [review]
try to fix a link issue

**Pull request merged**